### ISSUE TITLE
Itead Sonoff Touch

### ISSUE THREAD

**Issue by tam481** [arendst/Sonoff-MQTT-OTA-Arduino#40]
Hello, please add support for the newly released Sonoff Touch.

**Comment by Nayar** [comment]
Is it based on ESP8266 also? 

+1 for Sonoff Touch

**Comment by arendst** [comment]
Send me the hardware and I will try to implement it

**Comment by danjperron** [comment]
Works well with the sonoff touch. The only thing I had to do was to set the esp8265 and gnd one pin on power up.
[https://dl.dropboxusercontent.com/u/48891705/touch1.png](url)

**Comment by Slip7000** [comment]
Please mail me your Address so i can ship them to you! Treid to load the Touch and not working.

**Comment by danjperron** [comment]
I got 4 sonoff touch installed. I'm still waiting for 2 others + 1 sonoff and one S20 smart socket.

Try to,

1 -  Put the esp8265 to flash mode by shorting the pin like I show you on my previous post. Short the pin for 2 seconds on 3.3V power up.

2- Set the cpu to be esp8265.

3 - The switch is like the sonoff one.

B.T.W. I did use a esp12E with to relay and it works fine also. I did a video in french about it

https://youtu.be/T4XjFoNEUm8

This is my gig when I tried your code 

![img_20161127_192530](https://cloud.githubusercontent.com/assets/4349916/21082561/d61e2448-bfab-11e6-9dd5-8596ad635a07.jpg)

I'm planing to do a video using a C.H.I.P. + sonoff  + esp12E + mosquitto 1.4 QTT + paho.

![figure1](https://cloud.githubusercontent.com/assets/4349916/21082582/618ffdf8-bfac-11e6-910e-f5135d81f3b5.jpg)

Keep up the good work! I like what you did.
